tasks/digest-pinning.yml

```diff
@@ -18,8 +19,20 @@ spec:
         #! /usr/bin/env bash
         set -xe
 
+        if [ "$(params.skip)" == "true" ]; then
+          echo "Skip flag is set. Skipping digest pinning..."
+          echo "false" | tee $(results.dirty_flag.path)
+          exit 0
+        fi
+
         BUNDLE_PATH=$(realpath $(params.bundle_path))
         ls -l $BUNDLE_PATH
         operator-manifest pin $BUNDLE_PATH
 
-        echo "true" | tee $(results.dirty_flag.path)
+        if [[ $(git diff --stat) != '' ]]; then
+          echo "Manifests were not pinned."
```

Comment:
When I tried this condition on my computer it doesn't work properly. 

I tried to run this script in the repository with no changes:
```
if [[ $(git diff --stat) != '' ]]; then
  echo "Manifests were not pinned."
else
  echo "Manifests are pinned."
fi
```
Result- "Manifests are pinned.".